database/schema.sql

```diff
@@ -4,5 +4,6 @@ CREATE TABLE IF NOT EXISTS users (
     id          int NOT NULL UNIQUE AUTO_INCREMENT,
     firstName   varchar(20),
```

Comment:
We should probably add the firstName and lastName attributes to the User class as well if they are going in the DB.